templates/shortcode-receipt.php

```diff
@@ -150,12 +150,12 @@
 						<div class="edd_purchase_receipt_product_name">
 							<?php echo esc_html( $item['name'] ); ?>
 							<?php if( ! is_null( $price_id ) ) : ?>
-							<span class="edd_purchase_receipt_price_name">&nbsp;&ndash;&nbsp;<?php echo edd_get_price_option_name( $item['id'], $price_id, $payment->ID ); ?></span>
+							<span class="edd_purchase_receipt_price_name">&nbsp;&ndash;&nbsp;<?php echo esc_attr( edd_get_price_option_name( $item['id'], $price_id, $payment->ID ) ); ?></span>
```

Comment:
`esc_html()` would be appropriate except that breaks custom code where people have added HTML to the names.

It could use striptags or similar but would need to allow standard HTML